editor/src/utils/test-utils.ts

```diff
@@ -182,6 +183,13 @@ export function createFakeMetadataForParseSuccess(success: ParseSuccess): Array<
       elementMetadata = createFakeMetadataForJSXElement(
         component.rootElement,
         TP.scenePath([BakedInStoryboardUID, createSceneUidFromIndex(index)]),
+        {
+          props: {
+            style: props[PP.toString(PathForResizeContent)] ? props.style : undefined,
```

Comment:
Truthiness alert.

Reply:
Hmmm yes, good point. Will pull this out and make it not truthy

Reply:
Done (though it's still truthy, but it's a bit cleaner now)